src/sfctl/commands.py

```diff
@@ -212,6 +213,13 @@ def load_command_table(self, args): #pylint: disable=too-many-statements
 
         # Custom commands
 
+        with CommandSuperGroup(__name__, self,
+                               'sfctl.custom_container#{}',
+                               client_factory=client_create) as super_group:
+            with super_group.group('container') as group:
```

Comment:
This API is in grouping code package in swagger. Why call it something different here?

Reply:
Having a separate group is better as we will have more commands later and some of them are not tied to code packages. I can fix swagger later.

Reply:
Sure :) As long as we have parity with swagger eventually